Makefile

```diff
@@ -20,7 +20,7 @@ PythonModules = ["$(Namespace).$(ShortPackage)"]
 $(info PythonModules=${PythonModules})
 
 VFATQC_VER_MAJOR=2
-VFATQC_VER_MINOR=0
+VFATQC_VER_MINOR=1
 VFATQC_VER_PATCH=0
```

Comment:
FYI: changing the version in this way is no longer supported (since #184 though not yet implemented in this branch), and is instead picked up from a tag